tests/vexflow_test_helpers.js

```diff
@@ -65,6 +67,15 @@ VF.Test = (function () {
     // Default font properties for tests.
     Font: { size: 10 },
 
+    // Test all three fonts by default. Customize the FONTS_TO_TEST array to test fewer fonts.
+    FONTS_TO_TEST: DEFAULT_FONTS_TO_TEST,
```

Comment:
We should probably keep the semantics consistent and say `FONT_STACKS_TO_TEST: ['Bravura', 'Goneville', 'Petaluma']`. It's better to index by font stack than fonts.

Reply:
Sounds good.